### PR TITLE
fix(dashboard): refine keeper offline into unbooted/stopped

### PR DESCRIPTION
**Pull request by jeong-sik**
## Summary
- 백엔드가 보내는 generic `offline` 상태를 프론트엔드에서 keeper 메타데이터로 세분화
  - **미기동 (unbooted)**: 등록만 되고 한 번도 실행 안 됨 (generation=0, agent 없음)
  - **중단됨 (stopped)**: 이전에 활동했으나 현재 offline (generation>0)
  - **오프라인 (offline)**: 기존 fallback
- 각 상태에 고유 색상, 톤(warn/bad), 한글 레이블 지정
- `status-label.ts`의 dead-code 중복 (`stopped` → `문제`) 정리

## Changes
| 파일 | 변경 |
|------|------|
| `keeper-runtime-display.ts` | `refineOfflineStatus()` 함수 추가 |
| `status-label.ts` | `unbooted` → 미기동, `stopped` → 중단됨 (기존 중복 제거) |
| `tone.ts` | `stopped` → bad, `unbooted` → warn |
| `status-utils.ts` | `isOfflineStatus`에 새 상태 포함 |
| `keeper-detail.ts` | `statusColor` + `effectiveStatus`를 `keeperDisplayStatus()` 경유로 변경 |

## Test plan
- [x] TypeScript typecheck 통과
- [x] Vite build 성공 (중복 case 경고 해소)
- [x] 전체 테스트 315건 통과
- [ ] 실제 대시보드에서 미기동/중단됨/오프라인 구분 확인

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: dashboard/src/lib/keeper-runtime-display.ts [hunk 1]
```diff
@@ -4,9 +4,37 @@ import { relativeTime } from './format-time'
...
+
+  const generation = keeper.generation ?? 0
+  const turnCount = keeper.turn_count ?? 0
+  const agentExists = keeper.agent?.exists ?? false
+
+  // Never ran a single turn or generation — registered but never booted
+  if (generation === 0 && turnCount === 0 && !agentExists) {
+    return 'unbooted'
+  }
+
```
[the remark this answers is not in the document]
Comment (jeong-sik) [reply dashboard/src/lib/keeper-runtime-display.ts]: Acknowledged. This was merged without the tests. I will create a follow-up PR with unit tests for `refineOfflineStatus` covering both classification paths (unbooted vs stopped) and the edge cases.
